go/service/account.go

```diff
@@ -104,3 +104,65 @@ func (h *AccountHandler) ResetAccount(ctx context.Context, arg keybase1.ResetAcc
 
 	return h.G().Logout()
 }
+
+type GetLockdownResponse struct {
+	Enabled bool                       `json:"enabled"`
+	Status  libkb.AppStatus            `json:"status"`
+	History []keybase1.LockdownHistory `json:"history"`
+}
+
+func (r *GetLockdownResponse) GetAppStatus() *libkb.AppStatus {
+	return &r.Status
+}
```

Comment:
libkb.AppStatusEmbed is there to help

Reply:
wow we should change everything to use that